check.go

```diff
@@ -133,7 +133,7 @@ type CheckUpdateInput struct {
 	Level    *graphql.ID   `json:"levelId,omitempty"`
 	Owner    *graphql.ID   `json:"ownerId,omitempty"`
 	Filter   *graphql.ID   `json:"filterId,omitempty"`
-	Notes    string        `json:"notes,omitempty"`
+	Notes    *string       `json:"notes"`
```

Comment:
this doesn't actually need to be a pointer we just need to remove the `omitempty` to the "input" on the update mutation passes and empty string - the `omitempty` is droping the field "notes" from the variables on the mutation.